README.md

```diff
@@ -351,6 +351,11 @@ The error occurs because the credential that deploys the bundle doesn't have acc
 Contact your system administrator to make sure the `deploy_userid` configured for the CICS bundle deployment API has WRITE access to the bundles directory. The bundles directory is specified on the `com.ibm.cics.jvmserver.cmci.bundles.dir` option in the JVM profile of the CMCI JVM server.  
 For instructions on how to specify the bundles directory and grant access to `deploy_userid`, see [Configuring the CMCI JVM server for the CICS bundle deployment API](https://www.ibm.com/docs/en/cics-ts/5.6?topic=suc-configuring-cmci-jvm-server-cics-bundle-deployment-api) in CICS documentation.
 
+### Failed to execute goal com.ibm.cics:cics-bundle-maven-plugin:1.0.1-SNAPSHOT:deploy (default) on project standalone-war: Some of the supplied parameters were invalid
+**Why does it happen?**
+When you are using a bundle-war, bundle-ear, or bundle-osgi goal, the deploy goal must be configured with cics-bundle classifier specifically.
+**How to resolve it?** 
+Use the <classifier> configuration option to select the bundle, in the case where you're deploying a bundle-war-built bundle.
```

Comment:
i think we might as well be specific - they would be wanting `<classifier>cics-bundle</classifier>` - we might as well give them the exact string to be clear?

Reply:
done